### ISSUE TITLE
Trouble with the packaging tutorial

### ISSUE THREAD

**Issue by rafaelcarocaroquintero199-afk** [pypa/packaging-problems#1080]
Nombre RAFAEL PLACIDO BASURTO 
Remover definitivamente mis datos personales permanente de talk y algoritmo Skip to main content

scikit-learn homepage

API Reference
sklearn.neural_network
MLPClassifier
MLPClassifier
class sklearn.neural_network.MLPClassifier(hidden_layer_sizes=(100,), activation='relu', *, solver='adam', alpha=0.0001, batch_size='auto', learning_rate='constant', learning_rate_init=0.001, power_t=0.5, max_iter=200, shuffle=True, random_state=None, tol=0.0001, verbose=False, warm_start=False, momentum=0.9, nesterovs_momentum=True, early_stopping=False, validation_fraction=0.1, beta_1=0.9, beta_2=0.999, epsilon=1e-08, n_iter_no_change=10, max_fun=15000)
